pkg/registration/activation.go

```diff
@@ -0,0 +1,80 @@
+package registration
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/SUSE/connect-ng/pkg/connection"
+)
+
+// Activation holds information about a product activation.
+// subscription information.
+type Activation struct {
+	Name             string    `json:"name"`
+	Status           string    `json:"status"`
+	RegistrationCode string    `json:"regcode"`
+	Type             string    `json:"type"`
+	StartsAt         time.Time `json:"starts_at"`
+	ExpiresAt        time.Time `json:"expires_at"`
+
+	Metadata *Metadata
+	Product  *Product
+}
+
+type activationResponse struct {
+	Activation
+	MetadataAndProduct activateResponse `json:"service"`
+}
+
+var noActivations = []*Activation{}
```

Comment:
Do we really need this? As in, down below, wouldn't  `return []*Activation{}` work as well?

Reply:
no not necessarily. I'm just lazy, should I replace it?

Reply:
done!